src/pages/dashboard/pages/account/myActivities/myActivities.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import Post from './post';
+import Questions from './questions';
+import Answers from './answers';
+
+export default function MyActivities() {
+    const [render, setRender] = useState("Post")
+  return (
+    <main className="mx-auto w-[85%] pt-8">
+      <section className="">
+        <h2 className="text-medium mb-2 text-lg">Activities</h2>
+        <p className="text-medium text-sm text-neutral-600">
+          manage your posts, question, and other activities here
+        </p>
+      </section>
+      <section className="">
+        <article className="h-[3rem] w-full flex relative ">
+          <div
+            className="h-[inherit] relative w-max cursor-pointer text-center px-8 py-2 z-5"
```

Comment:
@vader-js TailwindCSS has a built-in `inherit` value for qualified properties. You're using it here and in other places in your code as an arbitrary value.

Reply:
Alright 
I’ll sort that out with the next update